### PR TITLE
Development

### PR DESCRIPTION
**Pull request by stratospherique**
this is a web page that we designed together in answer to the first assignment in the html and css curriculum. It's a youtube page. Your reviews are welcomed :)

### CODE REVIEW

File: assets/style.css [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+  vertical-align: middle;
+  }
+
+.icons {
+  height: 30px;
+  width: 30px;
+  vertical-align: middle;
+  margin: 5px;
+  position: relative;
+  left: 90px;
```
Comment (zlayabekrija) [inline assets/style.css]: in order to move icons to the right on the navbar, you should push them away from the input button, in your case list items and anchors are not in the same position. Also, add some display properties to the anchor.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+					</a>
+				</li>
+			</ul>
+		</nav>
+	</header>
+	<hr>
+	<main>
+		<div class="main">
+			<section class="col-2-3">
+				<iframe width="100%" height="415" src="https://www.youtube.com/embed/jOyfZex7B3E"
```
Comment (zlayabekrija) [inline index.html]: width in % is not allowed for iframe, use numbers instead .

File: index.html [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+									src="https://img.icons8.com/ios/50/000000/more.png"></a>
+						</li>
+					</ul>
+				</div>
+			</section>
+			<aside class="col-2-1">
+				<p>Related videos</p>
+				<ul class="related">
+					<li><a href=""><img src="https://images-na.ssl-images-amazon.com/images/I/91qahZOIs6L._SX342_.jpg"
+								alt="">
```
Comment (zlayabekrija) [inline index.html]: Please keep not more than one html tag per anchor.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+							<h3>Nintendo 64 | Gaming Historian</h3>
+							<br>
+							<h4>Views : Over 9000</h4>
+						</a></li>
+				</ul>
+			</aside>
+		</div>
+	</main>
+	<aside></aside>
```
Comment (zlayabekrija) [inline index.html]: Unnecessary please remove it.

**Review by zlayabekrija — CHANGES_REQUESTED** [review]
The first try was really good. There are some things to change still.
Every image needs to have an alt attribute.
Feel free to use fontawsome instead of the icons.
When you are ready run your code through https://validator.w3.org/ and preview it live on https://htmlpreview.github.io/ 
Follow up with notes on your code, please.
When you are ready submit the new review request and happy coding :)

File: index.html (hunk 2 above)
Comment (stratospherique) [reply index.html]: done :+1:

File: assets/style.css (hunk 1 above)
Comment (jofer86) [reply assets/style.css]: In order to achieve this, we placed an unnordered list inside a div, wich was the one that we pushed away from the search button, also we make sure, that the li, and the anchors were in the same position.

File: index.html (hunk 4 above)
Comment (stratospherique) [reply index.html]: removed :+1:

File: index.html (hunk 2 above)
Comment (jofer86) [reply index.html]: Fixed, we used pixels instead.

File: index.html (hunk 3 above)
Comment (jofer86) [reply index.html]: We removed the extra tags and left only one html tag per anchor.

File: index.html (hunk 4 above)
Comment (jofer86) [reply index.html]: They are gone now.

**Review by zlayabekrija — COMMENTED** [review]
Excellent job guys.
Fix these two things and you are good to go.

No need for new review request, just update this and you have my thumbs up

**Pull request merged**